File: packages/tasks-gen/snippets-fixtures/text-to-video/js/huggingface.js/0.fal-ai.js
import { InferenceClient } from "@huggingface/inference";

const client = new InferenceClient(process.env.HF_TOKEN);

const video = await client.textToVideo({
    provider: "fal-ai",
    model: "tencent/HunyuanVideo",
	inputs: "A young man walking on the street",
});
// Use the generated video (it's a Blob)

File: packages/tasks-gen/snippets-fixtures/text-to-video/python/huggingface_hub/0.fal-ai.py
from huggingface_hub import InferenceClient

client = InferenceClient(
    provider="fal-ai",
    api_key="api_token",
)

video = client.text_to_video(
    "A young man walking on the street",
    model="tencent/HunyuanVideo",
)